tests/src/symdump.ts

```diff
@@ -216,11 +216,10 @@ declare namespace Cypress {
     addDataSet(DSNAME: string, profileName: string): Chainable<any>;
   }
 }
-Cypress.Commands.add('addDataSet', (DSNAME, profileName) => {
+Cypress.Commands.add('addDataSet', (DSNAME, profileName, type) => {
```

Comment:
How is about `type = 'Batch'` for compatibility?

Reply:
I'm defining `type`  inside the test suite.

Reply:
Possible types are 'Batch' and 'CICS' - it is not a dataset type as it might look, it is type of Symdump service at the backend